src/util/image.rs

```diff
@@ -43,6 +45,13 @@ pub enum CacheBehavior {
     Skip,
 }
 
+#[derive(PartialEq)]
+pub enum ImageSearchBehaviour {
+    All,
+    // Valid means, that the image is properly attached to a review, be it unapproved or not
```

Comment:
We have no way of knowing that in the image service, since  - by design - we only know of images and where they reside (not if they are part of a valid review from the backend. I don't see what distinction we could make between "valid" and "all" images.

Reply:
Their state is encoded in the directory they are in. Especially for rotation, it does not make sense to search in the cached images, thus this distinction makes sense.

Pending images do not need to be considered for all operations which should only work on "actual" images. Pending ones may be deleted by the img service, other ones will not.

Reply:
I see, I did not look at the logic below and the wording confused me.